Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DPEComponentAdapter.cpp

```diff
@@ -168,10 +173,14 @@ namespace AZ::DocumentPropertyEditor
         ReflectionAdapter::CreateLabel(adapterBuilder, labelText, serializedPath);
     }
 
-    void ComponentAdapter::OnEntityDestroyed(const AZ::EntityId& entityId)
+    void ComponentAdapter::OnEntityDestruction(const AZ::EntityId& entityId)
     {
         if (entityId == m_entityId)
         {
+            AzToolsFramework::PropertyEditorGUIMessages::Bus::Handler::BusDisconnect();
+            AzToolsFramework::ToolsApplicationEvents::Bus::Handler::BusDisconnect();
+            AzToolsFramework::PropertyEditorEntityChangeNotificationBus::MultiHandler::BusDisconnect(m_entityId);
+            
```

Comment:
I'm wondering if some of these buses are expected to get called on entity destruction (ex. to update the outliner).